ontotext-yasgui-web-component/src/css/_variables.scss

```diff
@@ -7,3 +7,5 @@ $color-onto-white: #FFFFFF;
 $color-onto-grey: #00000059;
 $onto-blue-light: #0060b0;
 $background-onto-default: #FFF;
+
+$fullscreen-zet-index: 10000;
```

Comment:
This seems like to be too high value. I'm not sure why you exported such a constant. Although I think it was a good idea, I wonder if it wasn't possible to apply it though the top level dialog component. Also if it's a good idea to make many stacked containers to have one and the same z-index?!